config/sync/core.entity_form_display.media.embeddable.default.yml

```diff
@@ -0,0 +1,70 @@
+uuid: 9c7ec78e-9915-4755-8509-64cd31b8d0b8
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.media.embeddable.field_media_embeddable_code
+    - field.field.media.embeddable.field_media_embeddable_oembed
+    - media.type.embeddable
+  module:
+    - media
+    - path
+id: media.embeddable.default
+targetEntityType: media
+bundle: embeddable
+mode: default
+content:
```

Comment:
Hide all the fields that aren't necessary.

Reply:
fixed!